### PR TITLE
update smoke test case about go client

### PR DESCRIPTION
**Pull request by dleng-vmware**
Since the previous test code was so long ago, the current round of updates mainly clean up the previous test cases and add new test cases for the new features

### CODE REVIEW

**Review by junw-33455432 — APPROVED** [review]

File: autotest/client/go/goclienttest/TestCache/configCacheInitalService.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #singletonServer: http://localhost:8091
-singletonServer: https://localhost:8090
+singletonServer: https://g11n-vip-dev-1.eng.vmware.com:8090
```
Comment (junw-33455432) [inline autotest/client/go/goclienttest/TestCache/configCacheInitalService.yaml:2]: It seems the server can't be accessed via public network, please confirm.

File: autotest/client/go/goclienttest/TestCache/const.go [hunk 2]
```diff
@@ -38,4 +38,5 @@ const (
 	messagevalue     = "Your contact page."
 	frmessagevalue   = "Votre page de contact."
 	dlcnmessagevalue = "您的联系人页面。"
+	bat_path         = "E:/234/goclienttest/go_bat/"
```
Comment (junw-33455432) [inline autotest/client/go/goclienttest/TestCache/const.go:41]: It seems the bat_path is hard coded, please double confirm

File: autotest/client/go/goclienttest/TestCache/configCacheDisable.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #singletonServer: https://localhost:8090
-singletonServer: https://localhost:8090
+singletonServer: https://g11n-vip-dev-1.eng.vmware.com:8090
```
Comment (junw-33455432) [inline autotest/client/go/goclienttest/TestCache/configCacheDisable.yaml:2]: It seems the server can't be accessed via public network, please confirm.

File: autotest/client/go/goclienttest/TestCache/configCacheInitalServiceExpired.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #singletonServer: http://localhost:8091
-singletonServer: https://localhost:8090
+singletonServer: https://g11n-vip-dev-1.eng.vmware.com:8090
```
Comment (junw-33455432) [inline autotest/client/go/goclienttest/TestCache/configCacheInitalServiceExpired.yaml:2]: It seems the server can't be accessed via public network, please confirm.

**Review by junw-33455432 — CHANGES_REQUESTED** [review]
Hi Dabo, I've added some comments, please take a look.

File: autotest/client/go/goclienttest/TestCache/configCacheInitalService.yaml (hunk 1 above)
Comment (dleng-vmware) [reply autotest/client/go/goclienttest/TestCache/configCacheInitalService.yaml:2]: This is a configuration file that is no longer in use,  I will going to delete it

File: autotest/client/go/goclienttest/TestCache/configCacheInitalServiceExpired.yaml (hunk 4 above)
Comment (dleng-vmware) [reply autotest/client/go/goclienttest/TestCache/configCacheInitalServiceExpired.yaml:2]: This is a configuration file that is no longer in use,  I will going to delete it

File: autotest/client/go/goclienttest/TestCache/configCacheDisable.yaml (hunk 3 above)
Comment (dleng-vmware) [reply autotest/client/go/goclienttest/TestCache/configCacheDisable.yaml:2]: This is a configuration file that is no longer in use,  I will going to delete it

File: autotest/client/go/goclienttest/TestCache/const.go (hunk 2 above)
Comment (dleng-vmware) [reply autotest/client/go/goclienttest/TestCache/const.go]: the bat path not use in smoke test and i will change it to other path

**Review by junw-33455432 — APPROVED** [review]

**Pull request closed without merging**